include/libdnf/rpm/solv_query.hpp

```diff
@@ -406,13 +407,7 @@ class SolvQuery : public PackageSet {
     // TODO(jmracek) return std::pair<bool, std::unique_ptr<libdnf::rpm::Nevra>>
     /// @replaces libdnf/sack/query.hpp:method:std::pair<bool, std::unique_ptr<Nevra>> filterSubject(const char * subject, HyForm * forms, bool icase, bool with_nevra, bool with_provides, bool with_filenames);
     std::pair<bool, libdnf::rpm::Nevra> resolve_pkg_spec(
-        const std::string & pkg_spec,
-        bool icase,
-        bool with_nevra,
-        bool with_provides,
-        bool with_filenames,
-        bool with_src,
-        const std::vector<libdnf::rpm::Nevra::Form> & forms);
+        const std::string & pkg_spec, const libdnf::ResolveSpecSettings & settings, bool with_src);
```

Comment:
The `with_src` very much looks like it should be a part of `ResolveSpecSettings`.